controllers/teams.js

```diff
@@ -654,10 +661,12 @@ router.get('/:teamId', async (req, res, next) => {
 				directories,
 				files,
 				filesUrl: `/files/teams/${req.params.teamId}`,
+				nextcloudUrl,
+				useNextcloud,
 				ownerId: req.params.teamId,
-				canUploadFile: true,
-				canCreateDir: true,
-				canCreateFile: true,
+				canUploadFile: !useNextcloud,
```

Comment:
is ensured that all users that were able to upload files before have access to the nextcloud? giving the permission to school teachers is not enough here.

Reply:
We will give the nextcloud permission to all users with the role user in our next story